test/sample-suite.js

```diff
@@ -22,7 +22,6 @@ module.exports = {
           _retries: -1,
           _currentRetry: 0,
           pending: false,
-          type: 'test',
```

Comment:
Why was `type` removed? It is part of the raw test object so should be here. If you need to, add new objects for hooks with `type: 'hook'`.